docs/component-guides/profiles.md

```diff
@@ -1,68 +1,72 @@
 
-In order for the profile controller to interface with AWS IAM access control, the profiles controller pod must be recognized as an entity that can interface with IAM and be given the necessary permissions. This is done by using IAM Role Service Accounts (IRSA).
+For more information, see [Amazon EKS IAM roles for service accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html). 
 
-In AWS EKS, [IRSA](https://aws.amazon.com/blogs/opensource/introducing-fine-grained-iam-roles-service-accounts/) is a feature that allows enforcing AWS IAM permissions boundaries at the Kubernetes pod level. In a nutshell, a Kubernetes service account (SA) is associated with an IAM role with a role policy that scopes the IAM permissions (e.g. S3 read/write access, etc.). When a pod in the SA namespace is annotated with the SA name, EKS injects the IAM role arn and a token used to get the credentials so that the pod can make requests to AWS services within the scope of role policy associated with the IRSA.
+### Component-level implementations
 
-In order to enforce IAM permissions boundaries at the profile level, the profiles controller IRSA itself is not given the permissions required by the profiles. Instead, admins will create an IAM role for each profile and the profile controller IRSA will have permissions to access the admin created profile role and modify it to grant the `default-editor` SA in the profile namespsace permissions to assume the role. The `default-editor` SA in the profile namespace will also be annotated with the role arn. By doing both, EKS will then inject the profile role arn and a token into the pod in the profile namespace, allowing it to receive the credentials that grant it the permissions defined in the profile role policy.
+The following components have Profile-level support: 
+- Central Dashboard
+- Notebooks
+- Pipelines
+- AutoML (Katib)
+- KFServing
 
-The `default-editor` SA is used by various services in Kubeflow to launch resources in users namespace and hence the pods inherit the permissions of this service account. However, not all services do this by default.
+The following components have been tested to work with the `AwsIamForServiceAccount` plugin: 
+- Notebooks
 
-#### Component Level Implementations
+Integration with the `AwsIamForServiceAccount` plugin is being actively worked on for all components with Profile-level support. 
 
-As of writing, the following components have profile level support: Central Dashboard, Notebooks, Pipelines, AutoML (Katib), KFServing.
+You can find documentation about the `AwsIamForServiceAccount` plugin for specific components in the individual [component guides](/docs/component-guides/).
```

Comment:
using the plugin with various components will be found in different component guides. Information about plugin will be in this guide